config/config.go

```diff
@@ -36,6 +36,14 @@ var validIssueTrackers = []IssueTracker{
 
 var windowsAbsolutePathPattern = regexp.MustCompile("^[A-Z]{1}:")
 
+var originPatterns = map[IssueTracker]*regexp.Regexp{
+	IssueTrackerJira:    regexp.MustCompile(`[a-zA-Z0-9\-]+\.[a-zA-Z]+$`),
+	IssueTrackerGithub:  regexp.MustCompile(`^(https://)?(www\.)?github\.com/[a-zA-Z0-9\-_]+/[a-zA-Z0-9\-_]+`),
+	IssueTrackerGitlab:  regexp.MustCompile(`^(https://)?(www\.)?gitlab\.com/[a-zA-Z0-9\-_]+/[a-zA-Z0-9\-_]+`),
+	IssueTrackerPivotal: regexp.MustCompile(`^(https://)?(www\.)?pivotaltracker\.com/n/projects/[0-9]+`),
+	IssueTrackerRedmine: regexp.MustCompile(`[a-zA-Z0-9\-]+\.[a-zA-Z]+$`),
```

Comment:
This pattern excludes valid origins (i.e subdomains) such as `https://redmine.myorg.com`.

Please extend the pattern to allow such origins & add a test case for that.

Reply:
fixed in f4977d0 with test case in defa509